week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,9 +26,9 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  const isBigEnough = "Boolean";
```

Comment:
sorry for this being not clear. So usually when you read a variable name that comes as a question like , isItBig, isInfoRight, usually means it's a boolean.
Your work is correct. To follow the above-mentioned practice
```isBigEnough=num>=10```

Reply:
const isBigEnough=num>=10;